js/stores/appStore.js

```diff
@@ -439,10 +439,15 @@ const handleAppAction = (action) => {
           appState = appState.set('sites', siteUtil.addSite(appState.get('sites'), s, action.tag))
         })
       } else {
-        appState = appState.set('sites', siteUtil.addSite(appState.get('sites'), action.siteDetail, action.tag, action.originalSiteDetail))
+        let sites = appState.get('sites')
+        if (!action.siteDetail.get('folderId') && siteUtil.isFolder(action.siteDetail)) {
```

Comment:
This is the root cause of last backed out
https://github.com/brave/browser-laptop/commit/766722ad8e33706de2d53b6137e011ce233296a0#diff-23ca389e2bcb77191b5a9c10900eb3a3R443